src/js/view/controlbar.js

```diff
@@ -15,7 +16,9 @@ define([
         element.style.display = 'none';
 
         if (click) {
-            element.onclick = function() { click(); };
+            // Some actions receive parameters which change functionality based on their presence (E.G. play)
```

Comment:
apiAction adds some clarity.  The second comment explains why we aren't doing
``` javascript
new UI(element).on('click tap', apiAction);
```
instead since having the handler like that would send along the event as a parameter to the api call.